docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -364,40 +444,170 @@ Dagster Cloud currently includes support for four levels of role-based access co
       </td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>Edit workspace</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>
         <a href="/dagster-cloud/account/authentication">Administer SAML</a>
       </td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>View usage</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>Manage billing</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
   </tbody>
 </table>
+
+  </TabItem>
+</TabGroup>
+
+### Assigning user roles
+
+With the exception of the **Organization Admin** role, user roles are set on a per-deployment, per-code location basis and enforced both in Dagster Cloud and the GraphQL API. Organization Admins have access to the entire organization, including all [full deployments](/dagster-cloud/developing-testing/managing-deployments), [code locations](/dagster-cloud/developing-testing/code-locations), and [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments).
+
+#### Deployment permissions
+
+All Dagster Cloud accounts have the ability to set permissions at the deployment level. When defined, the selected user role will be used as the default for all code locations contained in the deployment.
```

Comment:
Only admins have the ability to set permissions - or does this mean both Standard and Enterprise plans do?

Reply:
Perhaps `plans` or something along those lines is better here. This is talking about what's available depending on the subscription you've chosen, not the user role you have.